src/pk-main.c

```diff
@@ -173,10 +173,14 @@ main (int argc, char *argv[])
 	/* get values from the config file */
 	conf = g_key_file_new ();
 	conf_filename = pk_util_get_config_filename ();
+	if (conf_filename == NULL) {
+		g_error ("Cannot found config file.");
```

Comment:
Yes. Well, we all agree that `g_error()` is not right.
Basically, `g_printerr()` -> stderr and `g_print()` -> stdout.
In practice, I agree. Maybe if someone would like to redirect part of the output (only errors).